### PR TITLE
Update doc of default functionals, closes #1218

### LINKED ISSUE #1218: Missmatch in KullbackLeibler functional and doc
The documentation says:

equation

while the implementation seems to implement

equation

while this may seem like a minor difference, it throws of computations where the scale is relevant.

Anyway, either we need to update the doc, or the implementation.

### PR DESCRIPTION
**Pull request by aringh**
Started as an attempt to close #1218. Ended up trying to removing all double backslashes from the documentation of the default functionals.

Comment: the change on line 186 is to make it less than 80 characters. Otherwise the sentence needs to be reformulated, and I was not able to come up with something good.

### CODE REVIEW

**Comment by adler-j** [comment]
The famous "Kullback L**Ö**eibler" functional! Except for that I'll get to work reviewing :) But please rebase the last 3 commits into one, easier to read.

File: odl/solvers/functional/default_functionals.py [hunk 1]
```diff
@@ -423,7 +423,7 @@ def __repr__(self):
 
 class IndicatorLpUnitBall(Functional):
 
-    """The indicator function on the unit ball in given the ``Lp`` norm.
+    r"""The indicator function on the unit ball in the :math:`L_p` norm.
```
Comment (adler-j) [inline odl/solvers/functional/default_functionals.py]: Avoid math in top comment, this should be readable (easily) without a fancy editor.

File: odl/solvers/functional/default_functionals.py [hunk 2]
```diff
@@ -1467,14 +1470,15 @@ def __repr__(self):
 
 
 class KullbackLeiblerCrossEntropyConvexConj(Functional):
-    """The convex conjugate of Kullback-Leibler Cross Entorpy functional.
+
+    r"""The convex conjugate of Kullback-Leibler Cross Entorpy functional.
```
Comment (adler-j) [inline odl/solvers/functional/default_functionals.py]: Convex conjugate of the Kullback-Leibler Cross Entorpy functional.

File: odl/solvers/functional/default_functionals.py [hunk 3]
```diff
@@ -1312,7 +1315,7 @@ def __repr__(self):
 
 class KullbackLeiblerCrossEntropy(Functional):
 
-    """The Kullback-Leibler Cross Entropy divergence functional.
+    r"""The Kullback-Leibler Cross Entropy divergence functional.
```
Comment (adler-j) [inline odl/solvers/functional/default_functionals.py]: ~The~ Kullback-Leibler Cross Entropy divergence functional.

File: odl/solvers/functional/default_functionals.py (hunk 2 above)
Comment (kohr-h) [reply odl/solvers/functional/default_functionals.py]: And fix the typo "Entorpy"

**Comment by aringh** [comment]
Same here, let's try to get this in :wink:. Good to go?

**Comment by aringh** [comment]
Most of this has been updated in later commit c90f3f984c1b976ac98675275242ea498ab061b7. I will close this and simply create a new PR for the update of the KL-doc.

**Pull request closed without merging**